language/documentation/book/src/abilities.md

```diff
@@ -0,0 +1,243 @@
+# Abilities
+
+Abilities are a typing feature in Move that control what actions are permissible for values of a given type. This system grants fine grained control over the "linear" typing behavior of values, as well as if and how values are used in global storage, by gating access to various byte code instructions. In order for a value to be used with the bytecode instruction, it must have the ability required (if one is required at all—not every instruction is gated by an ability).
+
+<!-- If you are already somewhat familiar with abilities from writing Move programs, but are still confused as to what is going on, it might be helpful to skip to the [motivating walkthrough](#motivating-walkthrough) section to get an idea of what the system is setup in the way that it is. -->
+
+## The Four Abilities
+
+The four abilities are:
+
+* [`copy`](#copy)
+    * Allows values of types with this ability to be copied
```

Comment:
I think each of these bullets should end with a period. Currently only one of them does.